trusted-committer/es/08-conclusion-es.asciidoc

```diff
@@ -0,0 +1,14 @@
+== Conclusión
+
+En capítulos anteriores hemos aprendido acerca de las responsabilidades de un Trusted Committer.
+Algunas de estas responsabilidades incluyen asegurar la calidad del producto, mantener una comunidad sana, reducir las barreras para hacer contribuciones, mejorando la comunidad y abogar por sus necesitades dentro de la compañia.
+También hemos hablado de como convertirse en un Trusted Committer y lo que se necesita para cumplir este rol.
+Trabajar como Trusted Committer es demandante pero al final va a amplificar el valor de tus contribuciones a la compañia.
+
+Nosotros esperamos haberte inspirado para que te embarques en el camino para convertirte en Trusted Committer.
+También esperamos haber ayudado a tu compañia a entender la importancia de tener Trusted Committers capaces para poder tener éxito en cualquier iniciativa InnerSource y el nível de empoderamiento que este rol requiere.
+
+Nos gustaría invitarte a aprender mas acerca de InnerSource al explorar otros artículos y videos en el camino de aprendizaje de InnerSource.
+Y por supuesto, estamos emocionados de darte la bienvenida en http://www.innersourcecommons.org/[la comunidad InnerSource].
+
+May the source be with you.
```

Comment:
This is a reference to a Star Wars' slogan "May the force be with you!", which in turn is inspired in "May god be with you". The spanish translation was "¡Que la fuerza te acompañe!". Thus it should read "Que el código te acompañe", but since the words aren't so similar as in english I'd suggest "_El código sea contigo_".

Reply:
I personally think there isn't a "good enough" translation and that it would be best to leave that part in english, but you have the final word :)

Reply:
No I don't. I arrived here just a few weeks ago. In this case any solution is correct and no one is clearly better. I prefer mine but I accept any.